pkg/ensurance/collector/collector.go

```diff
@@ -179,6 +180,13 @@ func (s *StateCollector) UpdateCollectors() {
 		if _, exists := s.collectors.Load(types.CadvisorCollectorType); !exists {
 			s.collectors.Store(types.CadvisorCollectorType, cadvisor.NewCadvisorCollector(s.podLister, s.GetCadvisorManager()))
```

Comment:
If the node resource feature disabled,  we can not collect the metric, can we add a judge for it